unit-tests/src/test/java/org/eclipse/collections/impl/factory/ListsTest.java

```diff
@@ -168,7 +168,7 @@ public void fixedSize()
     }
 
     @Test
-    public void multiReader()
+    public void implMultiReader()
     {
         MultiReaderListFactory listFactory = Lists.multiReader;
```

Comment:
In this test we are trying to test List.multiReader and org.eclipse.collections.api.factory.Lists.multiReader. Both reference  the same interface. The best way to do a comprehensive test for both is to invoke

this.testMultiReaderApi(List.multiReader);
this.testMultiReaderApi(org.eclipse.collections.api.factory.Lists.multiReader)

Thanks!!!!